snippets/integrations/README.md

```diff
@@ -0,0 +1,29 @@
+# Integration Code Snippets
+
+These are useful code snippets for common integration patterns
+
+# Installation
+
+If you are using VSCode on a mac you can install these snippets by placing them
+(or the whole file) at the following path
+`Users/$USER/Library/Application Support/Code/User/snippets/typescript.json`
+
+You can also edit the typescript snippets through the `Configure User Snippets`
+command in VSCode
+
+## List of snippets and usage
+
+Below you'll find all the available snippets and the usage of them.
+
+| Name                     | What it does?                                                                                                | How to use it?                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| ------------------------ | ------------------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --- |
```

Comment:
The rendered version of this document looks like it may have some issues. https://github.com/JupiterOne/sdk/blob/a1755c477f9ea1fa19e27c14256af489afdae11d/snippets/integrations/README.md

Reply:
Thanks for pointing that out, the new version renders it well. https://github.com/JupiterOne/sdk/blob/INT-6395-integration-snippets-docs/snippets/integrations/README.md